src/calculator.ts

```diff
@@ -0,0 +1,69 @@
+import fs from "fs";
+import { linearQF, Contribution, Calculation } from "pluralistic";
+
+type AugmentedResult = Calculation & {
+  projectName: string;
+  payoutAddress: string;
+};
+
+export default class Calculator {
+  private baseDataPath: string;
+  private chainId: string;
+  private roundId: string;
+
+  constructor(baseDataPath: string, chainId: string, roundId: string) {
+    this.baseDataPath = baseDataPath;
+    this.chainId = chainId;
+    this.roundId = roundId;
+  }
+
+  calculate() {
+    const rawContributions = this.parseJSONFile(
+      `${this.chainId}/rounds/${this.roundId}/votes.json`
+    );
+    const projects = this.parseJSONFile(`${this.chainId}/projects.json`);
+    const applications = this.parseJSONFile(
```

Comment:
`JSON.parse` could throw errors if the file is not found, has incorrect permissions, or has malformed JSON. We should consider handling the a thrown error here.